src/browser/components/MainPage.jsx

```diff
@@ -319,24 +343,40 @@ export default class MainPage extends React.Component {
     let tabsRow;
     if (this.props.teams.length > 1) {
       tabsRow = (
-        <Row>
-          <TabBar
-            id='tabBar'
-            teams={this.props.teams}
-            sessionsExpired={this.state.sessionsExpired}
-            unreadCounts={this.state.unreadCounts}
-            mentionCounts={this.state.mentionCounts}
-            unreadAtActive={this.state.unreadAtActive}
-            mentionAtActiveCounts={this.state.mentionAtActiveCounts}
-            activeKey={this.state.key}
-            onSelect={this.handleSelect}
-            onAddServer={this.addServer}
-            showAddServerButton={this.props.showAddServerButton}
-          />
-        </Row>
+        <TabBar
+          id='tabBar'
+          isDarkMode={this.state.isDarkMode}
+          teams={this.props.teams}
+          sessionsExpired={this.state.sessionsExpired}
+          unreadCounts={this.state.unreadCounts}
+          mentionCounts={this.state.mentionCounts}
+          unreadAtActive={this.state.unreadAtActive}
+          mentionAtActiveCounts={this.state.mentionAtActiveCounts}
+          activeKey={this.state.key}
+          onSelect={this.handleSelect}
+          onAddServer={this.addServer}
+          showAddServerButton={this.props.showAddServerButton}
+        />
       );
     }
 
+    let topBarClassName = 'topBar';
+    if (process.platform === 'darwin') {
+      topBarClassName += ' macOS';
+      if (this.state.isDarkMode) {
+        topBarClassName += ' darkMode';
```

Comment:
is this for debugging purposes?

Reply:
Actually, I forgot. It should be a `state` variable, since it can change as the OS changes from light to dark mode.